config/fund_loader_config/cof/eoi.py

```diff
@@ -0,0 +1,168 @@
+from datetime import datetime
+from datetime import timezone
+
+from config.fund_loader_config.cof.shared import COF_APPLICATION_GUIDANCE
+from config.fund_loader_config.common_fund_config.fund_base_tree_paths import (
+    COF_EOI_BASE_PATH,
+)
+from config.fund_loader_config.logo import DLUHC_LOGO_PNG
+
+COF_FUND_ID = "54c11ec2-0b16-46bb-80d2-f210e47a8791"
+COF_EOI_ROUND_ID = "6a47c649-7bac-4583-baed-9c4e7a35c8b3"
+
+APPLICATION_BASE_PATH_COF_EOI = ".".join([str(COF_EOI_BASE_PATH), str(1)])
+ASSESSMENT_BASE_PATH_COF_EOI = ".".join([str(COF_EOI_BASE_PATH), str(2)])
+
+COF_EOI_OPENS_DATE = datetime(2024, 3, 6, 11, 00, 0, tzinfo=timezone.utc)  # 2023-12-06 11:00:00
+COF_EOI_DEADLINE_DATE = datetime(2124, 3, 6, 11, 59, 0, tzinfo=timezone.utc)  # 2124-03-06 11:59:00
+COF_EOI_ASSESSMENT_DEADLINE_DATE = datetime(2124, 3, 6, 12, 0, 0, tzinfo=timezone.utc)  # 2124-03-06 12:00:00
+COF_EOI_SEND_REMINDER_DATE = datetime(2024, 3, 1, 11, 59, 0, tzinfo=timezone.utc)  # 2024-03-1 11:59:00
+
+
+fund_config = {
+    "id": COF_FUND_ID,
+    "name_json": {
+        "en": "Community Ownership Fund",
+        "cy": "Y Cronfa Perchnogaeth Gymunedol",
+    },
+    "title_json": {
+        "en": "an expression of interest in applying for Community",
+        "cy": "",
+    },
+    "short_name": "COF-EOI",
+    "description_json": {
```

Comment:
Can we grab this from `shared.py` so it isn't duplicated?

Reply:
So `shared.fund_config["description_json"]`

Reply:
Updated this to use shared description